docs/index.asciidoc

```diff
@@ -175,8 +175,9 @@ This plugin supports the following configuration options plus the <<plugins-{typ
 | <<plugins-{type}s-{plugin}-ssl_key_passphrase>> |<<password,password>>|No
 | <<plugins-{type}s-{plugin}-ssl_verify_mode>> |<<string,string>>, one of `["none", "peer", "force_peer"]`|No
```

Comment:
While we're making things right, will you please move the `ssl_verify_mode` option to its proper place in alpha order--both here and in the description?